packages/vapor/scss/icons/icons.scss

```diff
@@ -65,8 +65,8 @@
         &.mod-#{$size} {
             width: $size + px;
             height: $size + px;
-
-            @extend .mod-align-with-text;
+            line-height: $size + px;
+            vertical-align: -0.5 * ceil($size / 4) + px;
```

Comment:
By curiosity, how can we get this magic number?

Reply:
I ended up with this through trial and errors TBH